### PR TITLE
refactor: alert entity to include state of squad tour viewed

### PR DESCRIPTION
**Pull request by sshanzel**
Introduced a new alert property to identify whether to display the squad tour and be synced across devices.

### CODE REVIEW

File: src/entity/Alerts.ts [hunk 1]
```diff
@@ -18,6 +18,9 @@ export class Alerts {
   @Column({ type: 'bool', default: true })
   companionHelper: boolean;
 
+  @Column({ type: 'bool', default: true })
+  showSquadTour: boolean;
```
Comment (rebelchris) [inline src/entity/Alerts.ts:22]: Actually looking at the other we don't prefix `show` so maybe `squadTour` is enough?

**Review by rebelchris — APPROVED** [review]
Just one remark on naming

File: src/entity/Alerts.ts (hunk 1 above)
Comment (idoshamun) [reply src/entity/Alerts.ts:22]: Agree with this convention. Let's fix to `squadTour`

Comment (sshanzel) [reply src/entity/Alerts.ts:22]: Good point. Let me refactor it :ship

**Comment by sshanzel** [comment]
Have updated the property now.

@idoshamun should be ready for your review now 👀

**Review by rebelchris — APPROVED** [review]
Looks good!

**Review by idoshamun — APPROVED** [review]

**Pull request merged**